conf/settings.py

```diff
@@ -70,15 +74,16 @@
             # "connect_timeout": 28800,
         },
         # Persistent connections
-        "CONN_MAX_AGE": 0,
+        "CONN_MAX_AGE": None,
+        "CONN_HEALTH_CHECKS": True,
         # Disable server-side cursors, might be needed with pgbouncer
         "DISABLE_SERVER_SIDE_CURSORS": False,
     }
 }
 
 # Data directory, you can use following for the development purposes:
 # os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "data")
-DATA_DIR = os.path.join("__FINALPATH__", "data")
+DATA_DIR = "__FINALPATH__/data"
```

Comment:
Why don't we keep os.path.join?

Reply:
I thougt it was just changed by the updates settings file. What are the benefits?

Reply:
Hmm it works, just leave it like that, but I remember having issue with a malformed path (which should never happen)